tools/example-tool/myphasartool.cpp

```diff
@@ -17,9 +17,11 @@
 #include "phasar/PhasarLLVM/HelperAnalyses.h"
 #include "phasar/PhasarLLVM/Pointer/LLVMAliasSet.h"
 #include "phasar/PhasarLLVM/SimpleAnalysisConstructor.h"
+#include "phasar/PhasarLLVM/TypeHierarchy/DIBasedTypeHierarchy.h"
 #include "phasar/PhasarLLVM/TypeHierarchy/LLVMTypeHierarchy.h"
 
 #include <filesystem>
+#include <iostream>
```

Comment:
Please don't include `<iostream>`

See https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden and https://github.com/secure-software-engineering/phasar/wiki/Coding-Conventions